state/validation.go

```diff
@@ -144,6 +144,16 @@ func validateBlock(evidencePool EvidencePool, stateDB dbm.DB, state State, block
 				continue
 			}
 		}
+		// if we don't already have amnesia evidence we need to add it to start our own timer unless
+		// a) a valid polc has already been attached
+		// b) the accused node voted back on an earlier round
+		if ae, ok := ev.(types.AmnesiaEvidence); ok && ae.Polc.IsAbsent() && ae.PotentialAmnesiaEvidence.VoteA.Round <
+			ae.PotentialAmnesiaEvidence.VoteB.Round {
```

Comment:
`ae.PotentialAmnesiaEvidence.VoteA.Round < ae.PotentialAmnesiaEvidence.VoteB.Round`

can we do this in `ValidateBasic` func or something

Reply:
no, in this scenario we just want to know if the evidence we've never seen before can be executed straight away without having to wait for the trial period ourselves else we need to add it to the nodes evpool and go through the trial period.